src/assets/scss/_reset.scss

```diff
@@ -0,0 +1,99 @@
+@import "./mixin";
+* {
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+.w-100 {
+  width: $w_100;
+}
+.list-none {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+.word-break{
+  word-break: break-word;
+}
+.d-block {
+  display: block;
+}
+.d-none {
+  display: none;
+}
+.no-padding {
+  padding: 0 !important;
+}
+.no-margin {
+  margin: 0 !important;
+}
+input[type="submit"],
+button {
+  cursor: pointer;
+}
+img {
+  vertical-align: top;
+  border-style: none;
+}
+button:focus,
+textarea:focus,
+input:focus,
+select {
+  outline: none;
+}
+textarea,
+input[type="text"],
+input[type="email"],
+input[type="password"],
+input[type="tel"],
+input[type="number"],
+input[type="date"],
+select{
+  color:$main_color;
+}
+input[type="date"]::-webkit-calendar-picker-indicator{
+  background: $transparent;
+  color: $transparent;
+  cursor: pointer;
+  height: auto;
+  @include position_abs_left;
+  right: 0;
+  bottom: 0;
+  width: auto;
+}
+body {
+  margin: 0;
+  color: var(--body-text-color);
+  -webkit-text-size-adjust:  $w_100;
+  -ms-text-size-adjust: none;
+  font: 400 16px/20px "Roboto", sans-serif;

```

Comment:
in previous PR we have Poppins font family on body and in this PR we have Robot ??

Reply:
Now I have updated the font on both PR.